processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -349,6 +367,39 @@ public static HumanReadableBytes getAsHumanReadableBytes(
     throw badTypeException(key, "a human readable number", value);
   }
 
+  /**
+   * Insert, update or remove a single key to produce an overridden context.
+   * Leaves the original context unchanged.
+   *
+   * @param context context to override
+   * @param key     key to insert, update or remove
+   * @param value   if {@code null}, remove the key. Otherwise, inert or replace
+   *                the key.
+   * @return a new context map
+   */
+  public static Map<String, Object> override(
+      final Map<String, Object> context,
```

Comment:
don't see the `context` is used inside this function, do I miss sth?

Reply:
No, I did. Thanks for catching.